library/src/main/java/com/mapzen/android/dagger/AndroidModule.java

```diff
@@ -69,13 +69,21 @@
     TurnByTurnHttpHandler handler = new TurnByTurnHttpHandler();
     final String packageName = context.getPackageName();
     try {
-      final int apiKeyId = res.getIdentifier(API_KEY_TURN_BY_TURN_RES_NAME, API_KEY_RES_TYPE,
-          packageName);
+      final int apiKeyId = res.getIdentifier(ApiKeyConstants.API_KEY_TURN_BY_TURN_RES_NAME,
+          ApiKeyConstants.API_KEY_RES_TYPE, packageName);
       final String apiKey = res.getString(apiKeyId);
       handler.setApiKey(apiKey);
     } catch (Resources.NotFoundException e) {
       Log.e(TAG, e.getLocalizedMessage());
     }
     return handler;
   }
+
+  /**
+   * Provide initializer for configuring {@link MapzenSearch} objects.
+   * @return
+   */
+  @Provides @Singleton public SearchInitializer provideSearchInitializer() {
```

Comment:
Nice and clean. Perhaps we will want to introduce an initializer for tiles and routing at some point too similar to the `SearchInitializer` class.